src/views/PaymentNetwork.vue

```diff
@@ -74,54 +74,72 @@ function formatFiatCurrency(amount: number): string {
   return `${BASE_CURRENCY.symbol}${Math.round(amount).toLocaleString()}`;
 }
 
-function formatCryptoCurrency(amount: number, currency): string {
-  return `${amount.toLocaleString()} ${currency.code}`;
-}
-
 watch(paymentTx, () => {
   modalPostPaymentOpen.value = !!paymentTx.value;
 });
 </script>
 
 <template>
-  <h1>Network plan</h1>
-  <p class="text-md mb-4 leading-5">Pay to support your network on Snapshot</p>
+  <h1 class="leading-[44px] mb-[2px]">Network plan</h1>
+  <p class="mb-4 leading-[20px]">Pay to support your network on Snapshot</p>
 
   <form v-if="fxLoadStatus === 1" class="flex flex-col gap-4" @submit="pay">
     <fieldset>
-      <legend>
-        <h2 class="text-lg mb-2">Select your plan duration</h2>
+      <legend
+        class="text-[20px] mb-[12px] leading-none text-skin-heading font-semibold"
```

Comment:
```suggestion
        class="text-md mb-[12px] leading-none text-skin-heading font-semibold"
```
You can see text sizes in tailwind.config